### DISCUSSION TITLE
wiki changes

### DISCUSSION THREAD

**Discussion by xxcriticxx** [MajkiIT/polish-ads-filter#17367] [General] [answered]
@hawkeye116477 to jeszcze wiki trzeba pozmieniac

Jak stworzyć, opublikować i załadować kopię zapasową ustawień uBlocka Origin lub Nano Adblockera?
Jak blokować reklamy w przeglądarce Yandex na Androidzie (Nano Adblocker i Nano Defender, uBlock Origin i Nano Defender lub AdGuard Content Blocker)?
Jak zainstalować Nano Defender na Firefoksie, Waterfoksie, (Chr)Operze? albo Chrome?

**Comment by hawkeye116477** [comment]
@xxcriticxx Coś słabo patrzysz, 2 ostatnie już zostały zmienione w PR. 
Z tego co widzę, to co zostało, to w pierwszym, wzmianki o Nano do usunięcia i z tabelki z listami.

**Comment by xxcriticxx** [comment]
Skype 7 info można wykasować info jest z 2016

**Comment by hawkeye116477** [comment, marked as answer]
Ah, no fakt, dawno nie korzystałem ze Skype'a, teraz widzę, że u mnie tego pliku nie ma i reklam też nie, no chyba, że jedynie czasami się pojawiają.

**Reply by xxcriticxx** [reply]
kill kill kill

**Comment by krystian3w** [comment]
"urlHause" ma nowy dom w domenie `github.io`

**Reply by xxcriticxx** [reply]
thanks @hawkeye116477

**Comment by krystian3w** [comment]
@hawkeye116477 zaktualizujesz link Fanboya na jego hosting, coś nie umieją na `easylist-downloads.adblockplus.org` aktywować poprawnie:

`!#include bla-bla-car.txt`

**Comment by krystian3w** [comment]
Dla Firefox/PaleMoon/Basilisk trzeba chyba przebudować poradę: https://majkiit.github.io/polish-ads-filter/docs/basic/search-link-fix/

niby uAssets użył w "Annoyances": `set-constant, rwt, noopFunc` ale czy AG zaktualizowało też u siebie (niby nie), bo ABP/AdBlock pewnie tego nie dodał do swojej listy ze spinnetami.

> https://github.com/AdguardTeam/AdguardFilters/blob/69ed4fc3582e1d10f92b188d44cf4c96a1020f71/SpywareFilter/sections/general_extensions.txt#L38-L42

**Reply by krystian3w** [reply]
Dodatkowo na mobilnym raczej w ogóle nie  pomaga filtr uAssets.

**Reply by hawkeye116477** [reply]
Odnośnie Bazyliszka czy Księżyca, to tam jest problem taki, iż uBO Legacy nie obsługuje filtrów z gwiazdką (google.*). A rozszerzenia ClearURLs też nie ma dla tych przeglądarek, jest tylko takie coś https://addons.basilisk-browser.org/addon/pureurl4pm/, ale nie działa to na Google. Jedynie znalazłem działające ręczne rozwiązanie kopiuj/wklej/zamień.

**Comment by hawkeye116477** [comment]
Zapewne trzeba też usunąć Ungoogled Chromium z https://majkiit.github.io/polish-ads-filter/docs/advanced/android-browser-adblocker/#c-chromiumungoogled-chromium--uboublock-origin, bo się poddali i zrezygnowali z obsługi rozszerzeń. Za to najwyraźniej owocowa przeglądarka :kiwi_fruit: zrobiła wielki powrót.

**Reply by krystian3w** [reply]
O ile to nie łatki na 71 by udawała 92.

**Reply by hawkeye116477** [reply]
Żadne łatki, bazuje na Chromium 93. Chyba dodali też wsparcie dla dodatków ze sklepu Edge, tylko nie wiem komu miało by się to przydać, chyba raczej nie ma tam nic ciekawego.

**Comment by krystian3w** [comment]
Może to nie wiki ale link do PCC nie kieruje już poprawnie na sekcję "jak zainstalować" od momentu dodania angielszczyzny na wierzch a polszczyzny w katalogu "`/`"  i  średnio widocznym linkiem pod pierwszym nagłówkiem readme.

Niby można w repo PCC dodać `<span name="jak-zainstalowac"><span>` ale głupio to wygląda i nie jest zgodne z html5 jak i trzeba by to dać przed tekst z nagłówka "how to install" (a po `## `).

**Reply by hawkeye116477** [reply]
A gdzie taki link jest?

**Reply by krystian3w** [reply]
w tabelkach "polecamy" listy filtrów i może wyjątkowo też w jakieś poradzie wiki, gdy jakaś była o problemach z cookies/rodo.

**Reply by hawkeye116477** [reply]
No to w takiej sytuacji mówi się `Chase się tym zajmie` i dalej już wiesz co robić :smiley_cat:

Aktualizacja: #20392

**Comment by krystian3w** [comment]
Chyba jakiś css się wywala z Firefox:

![obraz](https://user-images.githubusercontent.com/35370833/165345209-1361def7-dc63-4c35-bbaa-61993702671b.png)

Ta czarna ramka wygląda dziwnie, 

hotfix:

```css
body[data-theme] td .dropdown-menu {
  background-color: transparent;
  border: transparent;
}
```



dodatkowo gitcdn.xyz zaczyna mulić - uBo wymieniło na gitcdn.link

**Reply by hawkeye116477** [reply]
Ależ nic się nie wywala. Niby tak ma być. W zasadzie oryginalnie w Bootstrapie jest na biało, a ja dopisałem ciemną odmianę, tylko może trochę mi nie wyszło. Pewnie najlepiej byłoby podmienić na gotowiec `https://github.com/ForEvolve/bootstrap-dark`.

**Comment by krystian3w** [comment]
Chyba gitloaf padł na dobre (ładuje się jakaś reklama a' la domeną zaparkowana lub przekierowanie w jakimś scam) i trzeba by szukać kolejnego mirrora/CDN.

**Reply by hawkeye116477** [reply]
Ta, wychodzi na to, że aktualnie pokazuje, iż strona została stworzona miesiąc temu, wykupiona przez DropCatch, czyli zapewne poprzedni właściciel nie opłacił przedłużenia, więc wykasowali z rejestru, a potem ten nowy przejął.
Powodzenia w szukaniu zastępstwa. Najwyraźniej każdy nowy jest na jakiś czas, a potem wygasa. Tak więc póki co zostają stare, ale jare Statically i JsDelivr.
Ewentualnie można wykorzystać Cloudflare Pages i zrobić własny mirror, o ile Cloudflare się nie obrazi, bo niby limitów łącza nie ma, jedyne ograniczenie to chyba 500x budowania na miesiąc. Gorhill i twórca Urlhaus tak robią.

**Comment by krystian3w** [comment]
Można by wymienić Alleblock na forka (najwyżej też umrze z braku czasu na projekty pro bono).

https://github.com/CrusheerPL/AlleBlockV2

https://raw.githubusercontent.com/CrusheerPL/AlleBlockV2/master/AlleBlockV2.txt (refs/heads w linku wydaje się opcjonalny).

Chyba że jest planowana jakaś impreza na szybkie scalanie PR przez majkela i xxcriticxxa.

**Reply by hawkeye116477** [reply]
Zrobione => https://github.com/MajkiIT/polish-ads-filter/pull/22512/commits/322980d5ca4e8bf57b0218d88ab6227dc0c1df65.
Tylko teraz problem jest taki, że najwyraźniej wszyscy z Białej Wieży są nie obecni i/lub brakuje im czasu.

**Comment by krystian3w** [comment]
Podobno Edge 134 dla Androida wspiera już uBlock Origin (niby nie Lite), ale nie wiadomo jak długo. Jak tylko Lite i inne MV3 z blokadą importu z CRX lub po ID to w sumie lipa:

> https://pl.reddit.com/r/browsers/comments/1eximvh/you_can_still_get_ublock_origin_on_edge_android/

Nie wiem czy mamy już wykreślony Kiwi 🥝 🐦‍⬛ z racji usunięcia go z Play Store (dla nowych użytkowników, starzy widzą i mogą reinstalować) i archiwizacji repozytorium - niby widzę go w artykule.

Magnolia zaczęła od jakiegoś czasu polecać chińskie [Quetta Browser](https://play.google.com/store/apps/details?id=net.quetta.browser) i niby są też jakieś inne próby tak jak Flowsurf i Mask Browser (jedna z dwóch niby nie wspiera filtrowania kart otwartych w tle).

**Reply by hawkeye116477** [reply]
Ale niby trzeba zmieniać język systemu na Chiński, a potem będzie problem jak wrócić do poprzednich ustawień.... Z tego co wiem, MS nie ma w planach usunąć MV2.

Jest więcej alternatyw: https://www.reddit.com/r/kiwibrowser/comments/1i1188s/kiwi_browser_alternatives_with_extension_support/, https://www.reddit.com/r/browsers/comments/1iabh56/comment/m9bjcc5
ale nie wiem czy są one warte sprawdzenia, pewnie z bardziej zaufanych zostaje jedynie Edge Canary albo fork lisa.

**Reply by krystian3w** [reply]
Niby są spekulacje, że jak Google zostanie sprzedane to upadnie Firefox.

**Reply by hawkeye116477** [reply]
A gdzie niby pisali, że Google jest na sprzedaż? A nawet gdyby zawsze mogą z inną firmą od wyszukiwarek się dogadać :smile:. No i tak w ogóle przecież kod jest otwarty, więc nawet gdyby Mozilla zrezygnowała, w każdej chwili ktoś może zrobić fork.

**Reply by hawkeye116477** [reply]
Na stabilnym Edge chyba najwyraźniej uBo już śmiga bez kombinacji. 
Jednakże wygląda na to, że powstaje też nowa przeglądarka na bazie Chromium z rozszerzeniami i wspiera sklep Opery i Google'a => https://github.com/gonzazoid/Ultimatum/releases.
Ja obecnie używam Iceraven, ale trochę lipa, że nie posiada importu i eksportu zakładek.
